plugins/pulp_rpm/plugins/importers/yum/parse/treeinfo.py

```diff
@@ -440,7 +442,7 @@ def process_distribution(self, tmp_dir):
                 CHECKSUM: None,
                 CHECKSUM_TYPE: None,
             })
-        return files
+        return files, result
```

Comment:
Because of this change the `:rtype:` and `:return:` of the `process_distribution` function needs to become a `tuple` with the positions documented in the docstring.